core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/request/ConceptMapMappingSearchRequestBuilder.java

```diff
@@ -53,12 +54,8 @@ public ConceptMapMappingSearchRequestBuilder filterByMapTargets(Iterable<String>
 		return addOption(OptionKey.MAP_TARGET, mapTargets);
 	}
 	
-	public ConceptMapMappingSearchRequestBuilder filterByComponentId(String componentId) {
-		return addOption(OptionKey.COMPONENT, componentId);
-	}
-	
-	public ConceptMapMappingSearchRequestBuilder filterByComponentIds(Iterable<String> componentIds) {
-		return addOption(OptionKey.COMPONENT, componentIds);
+	public ConceptMapMappingSearchRequestBuilder filterByComponentUri(ComponentURI uri) {
```

Comment:
I would add a String argument version as well, just to make it easier to filter without parsing into a bunch of `ComponentURI`s.

Reply:
But you still parse it into ComponentURIs. That is not actually necessary since you will be deserializing into a String to match the field in the index. Could you please use raw Strings instead? Thanks!